api.planx.uk/types.ts

```diff
@@ -12,3 +12,104 @@ export interface Flow {
     [key: string]: Node;
   };
 }
+
+export interface UserData {
+  answers?: Array<string>;
+  data?: Record<string, any>;
+  auto?: boolean;
+  feedback?: string;
+}
+
+export type Breadcrumb = Record<string, UserData>;
+
+export interface Team {
+  slug: string;
+  name: string;
+  domain?: string;
+  notifyPersonalisation: {
+    helpEmail: string;
+    helpPhone: string;
+    helpOpeningHours: string;
+    emailReplyToId: string;
+  };
+}
+
+export interface Passport {
+  data?: Record<string, any>;
+}
+
+export interface LowCalSession {
+  govUkPayment: any;
+  data: {
+    passport: Passport;
+    breadcrumbs: Breadcrumb;
+    govUkPayment: GovUKPayment;
+    id: string;
+  };
+  id: string;
+  created_at: string;
+  updated_at: string;
+  has_user_saved: boolean;
+  flow: {
+    slug: string;
+  };
+}
+
+export interface NotifyConfig {
+  personalisation: {
+    has_user_saved?: boolean | undefined;
+    address?: any;
+    projectType?: string;
+    id?: string;
+    expiryDate?: string;
+    helpEmail: string;
+    helpPhone: string;
+    helpOpeningHours: string;
+    emailReplyToId: string;
+    resumeLink?: string;
+    serviceLink?: string;
+    serviceName?: string;
+    teamName: string;
+  };
+  reference?: any;
+  emailReplyToId?: any;
+}
+
+// https://docs.payments.service.gov.uk/making_payments/#receiving-the-api-response
+export interface GovUKPayment {
```

Comment:
:+1:

one question: in the proxy response of our `/pay/:localAuthority/:paymentId` API endpoint, we'll return a filtered response that is only payment_id, amount, and state - should we mark other fields as optional here to account for that?